docs/openapi/components/schemas/credentials/GS1CompanyPrefixLicenseCredential.yml

```diff
@@ -0,0 +1,107 @@
+$linkedData:
+  term: GS1CompanyPrefixLicenceCredential
+  '@id': https://w3id.org/traceability#GS1CompanyPrefixLicenceCredential
+title: GS1 Company Prefix Licence Credential
```

Comment:
Is this meant to be US English or UK English?  Note that W3 intends to use US ...

If there's room for a comment somewhere, it might be good to note when UK English is intended for something (like perhaps this "Licence"), instead of being perceived as a misspelling (of "License", in this case).

Reply:
For certain, we should only use one of these (US or UK) in any given document. 

Optimally, we would only use one of these in any given repo.